src/App/App.js

```diff
@@ -0,0 +1,17 @@
+import { Routes, Route } from 'react-router-dom';
+import AllHorses from '../AllHorses/AllHorses';
+import Dashboard from '../Dashboard/Dashboard';
+import HorseProfile from '../HorseProfile/HorseProfile';
+import './App.css';
+
+function App() {
```

Comment:
Forgot to talk about how we wanted to organize the JS and SCSS files, but it looks like you settled on directories for each component, which works for me!